documentation/2-options.md

```diff
@@ -918,6 +918,35 @@ By default, requests will not use [method rewriting](https://datatracker.ietf.or
 
 For example, when sending a `POST` request and receiving a `302`, it will resend the body to the new location using the same HTTP method (`POST` in this case). To rewrite the request as `GET`, set this option to `true`.
 
+### `enableUnixSockets`
+
+**Type: `boolean`**\
+**Default: `true`**
+
+When it is enabled, requests can also be sent via [UNIX Domain Sockets](https://serverfault.com/questions/124517/what-is-the-difference-between-unix-sockets-and-tcp-ip-sockets).\
+Use the following URL scheme: `PROTOCOL://unix:SOCKET:PATH`
+
+By default it is enabled, so make sure you do your own URL sanitizing if you accept untrusted user input for the URL.
```

Comment:
Rewrite the reminder to highlight it is enabled by default.

Reply:
Can you make this a note like this: https://github.com/sindresorhus/got/blob/main/documentation/2-options.md#note-3